apis/beacon/states/validators.v2.yaml

```
@@ -0,0 +1,86 @@
+post:
+  operationId: "postStateValidatorsV2"
```

Same as https://github.com/ethereum/beacon-APIs/pull/452#discussion_r1656994933
```suggestion
  operationId: "getStateValidatorsV2"
```